server/GRPC_TA2_TA3/core_pb2.py

```diff
@@ -0,0 +1,1846 @@
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
Please remove the generated files from protoc. We will just generate them from ta2ta3-api/.